### PR TITLE
Fix file path for profiles

### LINKED ISSUE #21530: istioctl --installPackagePath not working like before - how to test local manifest changes?
**Bug description**

After making some changes to the manifests/, I'd like to locally test those changes, but since the monorepo work finished, the `istioctl --installPackagePath` option isn't working for me (I can't just point to my local repo anymore and get istioctl to install with my local changes).

How do you now test istioctl with local changes?

[ ] Configuration Infrastructure
[ ] Docs
[x ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**

Be able to run "istioctl" but yet have it pull in local changes to my manifests/ so I can test locally before submitting PRs.

**Steps to reproduce the bug**

Make changes to a manifest/, run `make gen` to do whatever else you need to submit a PR to make changes to the istioctl stuff.

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

1.5.0.beta3

**How was Istio installed?**

It isn't. That's what I'm trying to test - changes to istioctl manifests.

**Environment where bug was observed (cloud vendor, OS, etc)**

n/a

### PR DESCRIPTION
**Pull request by ostromart**
Fixes https://github.com/istio/istio/issues/21530.

### CODE REVIEW

File: operator/cmd/mesh/manifest-generate.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func GenManifests(inFilename []string, setOverlayYAML string, force bool,
 		return nil, nil, err
 	}
 	if err := cp.Run(); err != nil {
-		return nil, nil, fmt.Errorf("failed to create Istio control plane with spec: \n%v\nerror: %s", mergedIOPS, err)
```
Comment (elfinhe) [inline operator/cmd/mesh/manifest-generate.go]: I think this error is quite useful for debugging in-cluster operator (controller). Should we keep it a little longer?

**Review by elfinhe — COMMENTED** [review]
LGTM

File: operator/cmd/mesh/manifest-generate.go (hunk 1 above)
Comment (ostromart) [reply operator/cmd/mesh/manifest-generate.go]: the problem is it outputs a huge, unformatted blob of text and obscures the real error. we should have something here, but perhaps through --logtostderr, and with a formatted output?

Comment (ostromart) [reply operator/cmd/mesh/manifest-generate.go]: Error: failed to create Istio control plane with spec: 
profile:"/usr/local/google/home/mostrowski/go/src/istio.io/istio/operator/data/profiles/default.yaml" install_package_path:"/usr/local/google/home/mostrowski/go/src/istio.io/istio/manifests2" hub:"gcr.io/istio-testing" tag:latest components:<base:<enabled:<BoolValue:<value:true > > > pilot:<enabled:<BoolValue:<value:true > > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAME" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.name" > > > env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > readiness_probe:<httpGet:<path:"/ready" port:8080 > initialDelaySeconds:5 timeoutSeconds:5 periodSeconds:5 > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > policy:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-policy" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > telemetry:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > env:<name:"GOMAXPROCS" value:"6" > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-telemetry" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > replica_count:1 resources:<limits:<key:"cpu" value:"4800m" > limits:<key:"memory" value:"4G" > requests:<key:"cpu" value:"1000m" > requests:<key:"memory" value:"1G" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > citadel:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > galley:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<replica_count:1 resources:<requests:<key:"cpu" value:"100m" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > cni:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> > ingress_gateways:<enabled:<BoolValue:<value:true > > name:"istio-ingressgateway" tag:<nil> k8s:<hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-ingressgateway" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > resources:<limits:<key:"cpu" value:"2000m" > limits:<key:"memory" value:"1024Mi" > requests:<key:"cpu" value:"100m" > requests:<key:"memory" value:"128Mi" > > strategy:<rollingUpdate:<ma
...

Comment (elfinhe) [reply operator/cmd/mesh/manifest-generate.go]: sounds good if we change into log only.

**Review by elfinhe — APPROVED** [review]

**Comment by jmazzitelli** [comment]
Just documenting for posterity what @ostromart told me in slack re: how to use this:
```
note that if you're editing a profile locally in the repo, you have to use
--set profile=<path to your profile file> rather than --installPackagePath,
as the latter expects a full release tar structure, with both the charts and the profile yamls.
```

**Pull request merged**